README.md

```diff
@@ -283,6 +283,14 @@ Commands:
 
   patch <ID> <FILE>          Apply patch FILE to ID.
 
+  env-get                    Get the complete environment of the cluster master
```

Comment:
Would be more consistent with pmctl if it allowed getting keys by name, and had a `env` equivalent... but runctl is low-level tooling, ATM, not really intended for public use anymore, so perhaps we don't need niceties like that?